### ISSUE TITLE
Sometimes displays normal version of the page, not text

### ISSUE THREAD

**Issue by cru121** [dscorbett/view-link-as-text#2]
On the following page, many links open the regular version, not text.
https://addons.mozilla.org/en-US/firefox/addon/view-link-as-text/
For example, the "David Corbett" link, the "Add to Firefox" button, etc.
It's kinda ironic that the add-on does not work on the add-on page.

**Comment by dscorbett** [comment]
Have you noticed this on any domains besides addons.mozilla.org?

**Comment by cru121** [comment]
No, I have not noticed it elsewhere, but I'll come back here if I encounter anything.

**Comment by dscorbett** [comment]
In that case, I am pretty sure it is because Firefox restricts what you can do on addons.mozilla.org and testpilot.firefox.com, presumably to thwart malicious add-ons. I will try to suppress the “View Link as Text” option on those domains.
